src/testing/testhelpers/integ.py

```diff
@@ -85,8 +87,5 @@ def validate_schema(json_str, endpoint, validation_file):
     Validate a JSON response body against a schema.
     """
     schema = read_file(validation_file, endpoint.split('?', 1)[0])
-    try:
-        jsonschema.validate(json.loads(json_str), schema)
-    except jsonschema.exceptions.ValidationError as e:
-        return False
+    jsonschema.validate(json.loads(json_str), schema)
```

Comment:
Don't swallow the exception so that we get the details that which field fails the validation